lib/daylight.cpp

```diff
@@ -840,6 +859,9 @@ bool Daylight::simStandard(int blindGroupNum, int setting, Control *model){
         sunsOct=model->spaceDirectory()+model->intermediateDataDirectory()+model->spaceName()+"_"+model->windowGroups()[blindGroupNum].name()+"_sun_set"+std::to_string(setting+1)+"_std.oct";
 
     }
+    if (writeCL){
+            std::cout<<"***\nCreating suns octree.\n***"<<std::endl;
```

Comment:
OK, that's easy to do. For now, we can just have it be pretty much identical to what we're doing for info messages. Later we can add in conditional compilation stuff so that the messages don't appear when compiled in "release" mode.